src/status_im/data_store/chats.cljs

```diff
@@ -2,8 +2,18 @@
   (:require [goog.object :as object]
             [cljs.core.async :as async]
             [re-frame.core :as re-frame]
+            [status-im.utils.clocks :as utils.clocks]
             [status-im.data-store.realm.core :as core]))
 
+(defn- get-last-clock-value [chat-id]
```

Comment:
just wondering, is this a new function? can't see where it used

Reply:
mmh, you are right, I think something went funny with the rebase, let me check

Reply:
yes, good catch, I had extracted the fn but not used it :)

Reply:
updated!